packages/code-du-travail-frontend/src/outils/IndemnitePrecarite/steps/__tests__/situations.test.js

```diff
@@ -3,10 +3,9 @@ import {
   filterSituations,
   getNextQuestionKey,
   getOptions,
-  getPastQuestions,
-  validateSituation,
-  isNotYetProcessed
-} from "../situation";
+  getPastQuestions
+} from "../../../common/situations.utils";
```

Comment:
can you keep only test that are related to situation.js and move other test to situations.utils.test.js ?

Reply:
Done